apigee-cf-service-broker/api/service_instances.js

```diff
@@ -120,6 +120,38 @@ function actionValidate(req, res, next) {
   }
 }
 
+function deriveProtocol(params){
+  const proto = params.protocol ? params.protocol.toString().trim().toLowerCase() : ''
+  const ret = {
+    any: false,
+    errors: []
+  }
+  if (proto === "http" || proto === "https"){
+    ret["protocol"] = proto
```

Comment:
Bracket notation is used in `deriveAction` because the property name is from a variable. But there's only one constant property in this case, so dot notation would be clearer.